corehq/apps/locations/fixtures.py

```diff
@@ -60,6 +60,11 @@ def should_sync_locations(last_sync, location_db, restore_user):
 class LocationFixtureProvider(object):
     id = 'commtrack:locations'
```

Comment:
@snopoke I updated this fixture generator to optionally return two fixtures. as per the comment on https://github.com/dimagi/commcare-hq/blob/master/corehq/ex-submodules/casexml/apps/phone/fixtures.py#L33 it seems like I should change the `id` to just be `'commtrack'` based on that change, but was wondering exactly how the ID is used. `'commtrack'` is also a prefix for the product fixture and not sure how that would affect it.